tensorboard/util/util.py

```diff
@@ -63,6 +65,9 @@ def setup_logging(streams=(sys.stderr,)):
   werkzeug_logger = logging.getLogger('werkzeug')
   werkzeug_logger.setLevel(logging.WARNING)
   werkzeug_logger.handlers = handlers
+  absl_logger = logging.getLogger('absl')
+  absl_logging.set_verbosity(absl_logging.WARN)
+  absl_logger.handlers = handlers
```

Comment:
Do we still want to use our own log handler here (and for the cases above)?  I thought there was already the absl-provided one that gets installed at the root level.  So I'd actually have thought that with this arrangement we might get double logging - does that happen?

Unless our handler is much better than the standard absl one, I'd rather just use that one to simplify things.

Reply:
About what we had discussed offline: it looks like the argument/flag has to be parsed before we can determine whether the flag is set to default. This changes when we can configure logger (I am thinking putting it inside `program#configure` which is a bit larger scope to do in this PR. I will create a follow up PR for that.